### ISSUE TITLE
Unable to deploy Buffalo

### ISSUE THREAD

**Issue by jamtur01** [gobuffalo/buffalo#1854]
### Steps to Reproduce the Problem

Deploying a Buffalo app using:

  1. heroku container:push web or git push heroku master

### Expected Behavior

A dpeloyed app.

### Actual Behavior

```
Activity Feed  Build LogID 588918d6-f852-4525-bc19-e8c59ba76dbe
=== Fetching app code..
=== Building release (Dockerfile)
Sending build context to Docker daemon  522.8kBStep 1/20 : FROM gobuffalo/buffalo:v0.15.2 as builder
v0.15.2: Pulling from gobuffalo/buffalo
16ea0e8c8879: Pulling fs layer
50024b0106d5: Pulling fs layer
ff95660c6937: Pulling fs layer
9c7d0e5c0bc2: Pulling fs layer
2a19d2e6789c: Pulling fs layer
bd9cab5aeb31: Pulling fs layer
0ed236c95063: Pulling fs layer
bf0e01f4dc18: Pulling fs layer
9e3a56cb4f51: Pulling fs layer
03d3449452c0: Pulling fs layer
668b69cf6e2f: Pulling fs layer
167c6c2a18d2: Pulling fs layer
84d157e8c19a: Pulling fs layer
f9e1bf17ab34: Pulling fs layer
3515074fb3e1: Pulling fs layer
872efd12c105: Pulling fs layer
0f55b7f4a1a6: Pulling fs layer
6c6adef7705f: Pulling fs layer
ca5b6a81e533: Pulling fs layer
0d7c78bdbfb3: Pulling fs layer
3c5ec183bea3: Pulling fs layer
465edd9bddd1: Pulling fs layer
9c7d0e5c0bc2: Waiting
2a19d2e6789c: Waiting
bd9cab5aeb31: Waiting
0ed236c95063: Waiting
bf0e01f4dc18: Waiting
9e3a56cb4f51: Waiting
03d3449452c0: Waiting
668b69cf6e2f: Waiting
167c6c2a18d2: Waiting
84d157e8c19a: Waiting
f9e1bf17ab34: Waiting
3515074fb3e1: Waiting
872efd12c105: Waiting
3c5ec183bea3: Waiting
0f55b7f4a1a6: Waiting
6c6adef7705f: Waiting
ca5b6a81e533: Waiting
0d7c78bdbfb3: Waiting
465edd9bddd1: Waiting
ff95660c6937: Verifying Checksum
ff95660c6937: Download complete
9c7d0e5c0bc2: Verifying Checksum
9c7d0e5c0bc2: Download complete
16ea0e8c8879: Verifying Checksum
16ea0e8c8879: Download complete
50024b0106d5: Verifying Checksum
50024b0106d5: Download complete
0ed236c95063: Verifying Checksum
0ed236c95063: Download complete
2a19d2e6789c: Verifying Checksum
2a19d2e6789c: Download complete
bf0e01f4dc18: Verifying Checksum
bf0e01f4dc18: Download complete
03d3449452c0: Verifying Checksum
03d3449452c0: Download complete
bd9cab5aeb31: Verifying Checksum
bd9cab5aeb31: Download complete
668b69cf6e2f: Verifying Checksum
668b69cf6e2f: Download complete
167c6c2a18d2: Verifying Checksum
167c6c2a18d2: Download complete
84d157e8c19a: Verifying Checksum
84d157e8c19a: Download complete
f9e1bf17ab34: Verifying Checksum
f9e1bf17ab34: Download complete
9e3a56cb4f51: Verifying Checksum
9e3a56cb4f51: Download complete
3515074fb3e1: Download complete
872efd12c105: Verifying Checksum
872efd12c105: Download complete
0f55b7f4a1a6: Verifying Checksum
0f55b7f4a1a6: Download complete
0d7c78bdbfb3: Verifying Checksum
0d7c78bdbfb3: Download complete
ca5b6a81e533: Verifying Checksum
ca5b6a81e533: Download complete
6c6adef7705f: Verifying Checksum
6c6adef7705f: Download complete
465edd9bddd1: Verifying Checksum
465edd9bddd1: Download complete
16ea0e8c8879: Pull complete
3c5ec183bea3: Verifying Checksum
3c5ec183bea3: Download complete
50024b0106d5: Pull complete
ff95660c6937: Pull complete
9c7d0e5c0bc2: Pull complete
2a19d2e6789c: Pull complete
bd9cab5aeb31: Pull complete
0ed236c95063: Pull complete
bf0e01f4dc18: Pull complete
9e3a56cb4f51: Pull complete
03d3449452c0: Pull complete
668b69cf6e2f: Pull complete
167c6c2a18d2: Pull complete
84d157e8c19a: Pull complete
f9e1bf17ab34: Pull complete
3515074fb3e1: Pull complete
872efd12c105: Pull complete
0f55b7f4a1a6: Pull complete
6c6adef7705f: Pull complete
ca5b6a81e533: Pull complete
0d7c78bdbfb3: Pull complete
3c5ec183bea3: Pull complete
465edd9bddd1: Pull complete
Digest: sha256:42eb7fdca8bd30b42fbfe260c04268cc671c1c4ace57840253984d5e03a08b20
Status: Downloaded newer image for gobuffalo/buffalo:v0.15.2
 ---> c4fbcd773577
Step 2/20 : RUN mkdir -p $GOPATH/src/coinage
 ---> Running in 3ea8d8f046c4
Removing intermediate container 3ea8d8f046c4
 ---> 8f02529e3df3
Step 3/20 : WORKDIR $GOPATH/src/coinage
 ---> Running in 631847d96ef4
Removing intermediate container 631847d96ef4
 ---> 6b1ccf0dcfa6
Step 4/20 : ADD package.json .
 ---> d651fdcc9116
Step 5/20 : ADD yarn.lock .
 ---> e3f2b4d25b87
Step 6/20 : RUN yarn install --no-progress
 ---> Running in 49e2a2edaefd
yarn install v1.19.2
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
Done in 10.67s.
Removing intermediate container 49e2a2edaefd
 ---> 63288b3e87b6
Step 7/20 : ADD . .
 ---> 63ff93862b94
Step 8/20 : ENV GO111MODULES=on
 ---> Running in 54d70c7ff89a
Removing intermediate container 54d70c7ff89a
 ---> 6d764682c3f9
Step 9/20 : RUN go get ./...
 ---> Running in 0a731a02681d
# github.com/gobuffalo/buffalo/plugins/plugdeps
../github.com/gobuffalo/buffalo/plugins/plugdeps/plugin.go:15:11: undefined: meta.BuildTags
The command '/bin/sh -c go get ./...' returned a non-zero code: 2
```

### Info

<details>

```
-> Go: Checking installation
✓ The `go` executable was found on your system at: /usr/local/bin/go

-> Go: Checking minimum version requirements
✓ Your version of Go, 1.13.4, meets the minimum requirements.

-> Go: Checking GOPATH
✓ You are using Go Modules, so no need to worry about the GOPATH.

-> Go: Checking Package Management
✓ You are using Go Modules (`go`) for package management.

-> Go: Checking PATH
✓ Your PATH contains /Users/james/go/bin.

-> Node: Checking installation
✓ The `node` executable was found on your system at: /usr/local/bin/node

-> Node: Checking minimum version requirements
✓ Your version of Node, v11.6.0, meets the minimum requirements.

-> NPM: Checking installation
✓ The `npm` executable was found on your system at: /usr/local/bin/npm

-> NPM: Checking minimum version requirements
✓ Your version of NPM, 6.13.1, meets the minimum requirements.

-> Yarn: Checking installation
✓ The `yarnpkg` executable was found on your system at: /usr/local/bin/yarnpkg

-> Yarn: Checking minimum version requirements
✓ Your version of Yarn, 1.19.1, meets the minimum requirements.

-> PostgreSQL: Checking installation
✓ The `postgres` executable was found on your system at: /usr/local/bin/postgres

-> PostgreSQL: Checking minimum version requirements
✓ Your version of PostgreSQL, 12.1.0, meets the minimum requirements.

-> MySQL: Checking installation
✓ The `mysql` executable was found on your system at: /usr/local/bin/mysql

-> MySQL: Checking minimum version requirements
✓ Your version of MySQL, 8.0.18, meets the minimum requirements.

-> SQLite3: Checking installation
✓ The `sqlite3` executable was found on your system at: /usr/bin/sqlite3

-> SQLite3: Checking minimum version requirements
✓ Your version of SQLite3, 3.28.0, meets the minimum requirements.

-> Cockroach: Checking installation
✘ The `cockroach` executable could not be found on your system.
For help setting up your Cockroach environment please follow the instructions for you platform at:

https://www.cockroachlabs.com/docs/stable/

-> Buffalo (CLI): Checking installation
✓ The `buffalo` executable was found on your system at: /Users/james/go/bin/buffalo

-> Buffalo (CLI): Checking minimum version requirements
✘ Invalid Semantic Version

-> Buffalo: Application Details
Pwd         /Users/james/Dropbox/go/src/github.com/jamtur01/coinage
Root        /Users/james/Dropbox/go/src/github.com/jamtur01/coinage
GoPath      /Users/james/go
PackagePkg  coinage
ActionsPkg  coinage/actions
ModelsPkg   coinage/models
GriftsPkg   coinage/grifts
WithModules true
Name        coinage
Bin         bin/coinage
VCS         git
WithPop     true
WithSQLite  false
WithDep     false
WithWebpack true
WithNodeJs  true
WithYarn    true
WithDocker  true
WithGrifts  true
AsWeb       true
AsAPI       false
InApp       true
PackageJSON {map[build:webpack -p --progress dev:webpack --watch]}

-> Buffalo: config/buffalo-app.toml
name = "coinage"
bin = "bin/coinage"
vcs = "git"
with_pop = true
with_sqlite = false
with_dep = false
with_webpack = true
with_nodejs = true
with_yarn = true
with_docker = true
with_grifts = true
as_web = true
as_api = false

-> Buffalo: config/buffalo-plugins.toml
[[plugin]]
  binary = "buffalo-heroku"
  go_get = "github.com/gobuffalo/buffalo-heroku"

[[plugin]]
  binary = "buffalo-pop"
  go_get = "github.com/gobuffalo/buffalo-pop"

-> Buffalo: go.mod
module coinage

go 1.12

require (
	github.com/aws/aws-sdk-go v1.25.43
	github.com/cockroachdb/cockroach-go v0.0.0-20190925194419-606b3d062051 // indirect
	github.com/gobuffalo/buffalo v0.15.2
	github.com/gobuffalo/buffalo-pop v1.23.1
	github.com/gobuffalo/envy v1.8.1
	github.com/gobuffalo/here v0.5.1 // indirect
	github.com/gobuffalo/meta/v2 v2.0.0 // indirect
	github.com/gobuffalo/mw-csrf v0.0.0-20190129204204-25460a055517
	github.com/gobuffalo/mw-forcessl v0.0.0-20190224202501-6d1ef7ffb276
	github.com/gobuffalo/mw-i18n v0.0.0-20190224203426-337de00e4c33
	github.com/gobuffalo/mw-paramlogger v0.0.0-20190224201358-0d45762ab655
	github.com/gobuffalo/nulls v0.1.0
	github.com/gobuffalo/packr v1.30.1 // indirect
	github.com/gobuffalo/packr/v2 v2.7.1
	github.com/gobuffalo/pop v4.12.2+incompatible
	github.com/gobuffalo/suite v2.8.1+incompatible
	github.com/gobuffalo/validate v2.0.3+incompatible
	github.com/gobuffalo/x v0.1.0
	github.com/gofrs/uuid v3.2.0+incompatible
	github.com/golang/protobuf v1.3.2 // indirect
	github.com/jackc/pgconn v1.1.0 // indirect
	github.com/markbates/grift v1.5.0
	github.com/mattn/go-colorable v0.1.4 // indirect
	github.com/mattn/go-isatty v0.0.10 // indirect
	github.com/mattn/go-sqlite3 v2.0.0+incompatible // indirect
	github.com/pelletier/go-toml v1.6.0 // indirect
	github.com/pkg/errors v0.8.1
	github.com/rogpeppe/go-internal v1.5.0 // indirect
	github.com/spf13/afero v1.2.2 // indirect
	github.com/spf13/jwalterweatherman v1.1.0 // indirect
	github.com/unrolled/secure v1.0.1
	golang.org/x/crypto v0.0.0-20191128160524-b544559bb6d1
	golang.org/x/net v0.0.0-20191126235420-ef20fe5d7933 // indirect
	golang.org/x/sys v0.0.0-20191128015809-6d18c012aee9 // indirect
	golang.org/x/tools v0.0.0-20191130070609-6e064ea0cf2d // indirect
	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898 // indirect
	google.golang.org/appengine v1.6.5 // indirect
)

replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190409202823-959b441ac422

replace sourcegraph.com/sourcegraph/go-diff v0.5.1 => github.com/sourcegraph/go-diff v0.5.1

replace golang.org/x/lint => github.com/golang/lint v0.0.0-20190930215403-16217165b5de
```

</details>

**Comment by jamtur01** [comment]
Regarding the semantic version error - it fails on 0.15.0 and 0.15.2 - tried an upgrade unsuccessfully.

**Comment by markbates** [comment]
There’s a typo in your Dockerfile. GO111MODULES should be GO111MODULE

**Comment by H3rby7** [comment]
@markbates I also stumbled upon this issue. The Dockerfile created by `buffalo new` also has this typo included -> 

see: https://github.com/H3rby7/kebap/blob/master/Dockerfile

which is freshly created via buffalo new

**Comment by jamtur01** [comment]
Yes - I was coming back to say that I hadn't edited the Dockerfile so it must be somewhere upstream. :)

**Comment by jamtur01** [comment]
c.f. #1839

**Comment by jamtur01** [comment]
Which upon further research appears to be fixed in https://github.com/gobuffalo/buffalo/commit/8bb3200c78a83a7d5754cbd9e630e081098a8cb8#diff-a8318fa355ef1f7f99de7c43dbc9217b

**Comment by withakay** [comment]
Despite the fix above, that appears to be merged into 15.3 (?), I just got caught by this in a brand new project...
